buildenv/jenkins/common/pipeline-functions

```diff
@@ -28,29 +28,46 @@
     // if not set, sniff the remote repositories references
 
     stage ('Sniff Repos') {
-        dir('openjdk') {
+        def description = ''
+
+        if (OPENJDK_SHA instanceof Map && OPENJDK_REPO instanceof Map && OPENJDK_BRANCH instanceof Map) {
+            // fetch SHAs for all OpenJDK repositories
+            def OPENJDK_SHAS_BY_RELEASES = [:]
+
+            OPENJDK_SHA.each { release, sha ->
```

Comment:
I cannot see a use case where this loop is ever executed

Reply:
see variables-functions.set_repos_variables().